### ISSUE TITLE
$tree.tree('openNode', node) not working

### ISSUE THREAD

**Issue by Chocapic** [mbraak/jqTree#353]
I have a tree with multiple selection enabled. When I select a node I would like it to be opened and add all his children to the selection. 

Im doing as follow :

```javascript
    $tree.bind('tree.click', function (e) {
        e.preventDefault();
        var node = e.node;
        var parent = node.parent;

        if (!$tree.tree('isNodeSelected', node)) { 
            $tree.tree('addToSelection', node); 
            $tree.tree('opendNode', node);

            node.iterate(function (child_node) { 
                $tree.tree('addToSelection', child_node);
                return true;
            });
  });
```

The multiple selection works fine however the selected node is not opening even if he has children. 

On a related note : I have a function that returns all final selected node (node selected that don't have children) as follow :

```javascript
function getSelectedNodesValueV($tree) {
    var res = [];
    var node = $tree.tree('getNodeById', 1); //start at first node
    while (node) { //tant que pas null
        if ($tree.tree('isNodeSelected', node)) {
            if (node.children.length == 0) { //if don't have children
                res.push(node.value);
            }
        }
        node = node.getNextNode();
    }
    return res;
}
```

this works fine except that if a node is selected (and thus all his chldren selected aswell) but is closed, the function returns null. How could I fix that ?

Thanks a lot

**Comment by mbraak** [comment]
I think you have a typo in your code: it should be ``openNode`` instead of ``opendNode``.

Cheers

**Comment by mbraak** [comment]
About the second question: perhaps you can use the [getSelectedNodes](http://mbraak.github.io/jqTree/#multiple-selection-get-selected-nodes) function.

**Comment by Chocapic** [comment]
Thanks for the typo, it was driving me crazy. About getSelectedNodes the only issue I have with this function is that if you have a node (e.g. node 1) and node 1 has children (node 1-1 and node 1-2) :

when I selected node 1 with the algorithm above its children get selected aswell. getSelectedNode will return [node 1, node 1-1, node 1-2] when I would like to only get node 1-1 and node 1-2 (exlude a parents from the selection if all his children are selected). That's why I wrote this piece of code which exclude parents.
